release_notes.md

```diff
@@ -2,6 +2,9 @@
 
 ### Common
 
+* Added `MultivariateGaussianProposalDistribution`. It assumes a diagonal covariance matrix.
+  * Example in Python:
+    * `MetropolisHastingsSampler(proposal_distribution='multivariate_gaussian', latents=[v1, v2], proposal_distribution_sigma=[1., np.array([[1., 2.], [3., 4.]]))`
```

Comment:
Do we need to add any of this to the shinydocs?

Reply:
It'll be in Python docs